h/api/test/schemas_test.py

```diff
@@ -42,13 +42,13 @@ def test_it_sets_appropriate_error_message_when_data_invalid(self):
 class TestAnnotationSchema(object):
```

Comment:
Hmm. Not so sure. The JSON schema is a data structure. I'm not sure what a test saying "field foo is required" adds over a line in a JSON schema that says `required: True`.

Even if we think the tests add value, they're currently incredibly verbose and coupled to various details of jsonschema, such as error message formats.

Not for this PR, but if anything this should be a big list of failing cases with comments next to them, run through one test with `pytest.mark.parametrize`.